src/EvmToCairo.py

```diff
@@ -38,15 +39,14 @@
    local stack0 : StackItem
    assert stack0 = StackItem(value=Uint256(-1, 0), next=&stack0)  # Points to itself.
 
-   let (local res) = run_from{{msize=msize, memory_dict=memory_dict}}(Uint256(0, 0), &stack0)
-   default_dict_finalize(memory_start, memory_dict, 0)
```

Comment:
Why remove dict finalization?

Reply:
It doesn't work on StarkNet

Reply:
I think, Shahar has mentioned that they've whitelisted dict hints, so it should work now?..

Reply:
default_dict_finalize isn't whitelisted yet.